comp/core/workloadmeta/impl/component_mock.go

```diff
@@ -21,31 +21,33 @@ import (
 
 // Mock implements mock-specific methods.
 type Mock interface {
```

Comment:
This should be in the `mock` package 😄 Same we did for the replay component https://github.com/DataDog/datadog-agent/pull/25512

Reply:
`MockModuleV2` wraps the actual impl, so I would need to export a lot of things, for now I'd rather merge the PR as is (it's already way too big), but I'm thinking of having a follow-up to clean what's needed properly